index.js

```diff
@@ -7,64 +7,59 @@ const io = require('socket.io')(http);
 const schedulerDataService = require('./data-services/scheduler-data-service.js');
 const battleGoalDataService = require('./data-services/battle-goals-data-service.js');
 
-async function main() {
-  app.use(express.static('app'));
-
-  await initializeScheduler();
-  await initializeBattleGoals();
-
-  http.listen(3000, function() {
-    console.log('listening on port: 3000');
-  });
+function initializeCron() {
+  const job = new CronJob('0 0 0 1 * *', function() {
+    schedulerDataService.nextMonth();
+    io.emit('weeks', schedulerDataService.get().weeks);
+  }, null, true, 'America/New_York');
+  job.start();
```

Comment:
the io object in here isnt in the scheduler namespace. the scheduler wont receive this message

Reply:
either use .of('/scheduler') or put this in the initialize scheduler function